exercises/concept/hyperia-forex/HyperiaForexTests.cs

```diff
@@ -22,8 +22,6 @@ public void Equality_bad()
         Assert.Throws<ArgumentException>(() => Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(60, "USD")));
     }
 
-    private bool Eq(CurrencyAmount amountA, CurrencyAmount amountB)
-    {
-        return amountA == amountB;
-    }
+    private bool Eq(CurrencyAmount amountA, CurrencyAmount amountB) => amountA == amountB;
```

Comment:
I'm sorry, but I didn't mean make it an expression body, but to inline the method. So instead of `Eq(new CurrencyAmount(55, "HD"), new CurrencyAmount(60, "USD"))` to `new CurrencyAmount(55, "HD") == new CurrencyAmount(60, "USD")`. The `Eq` method should be gone by the end.

Reply:
Hi, 
Thanks for your review, hope my latest commit fixes the problem